src/main/resources/public/js/app.js

```diff
@@ -53,7 +53,7 @@ function reload() {
             }
              
             function sendMessage(msg) {
-                stompClient.send("/notifications", {},
+                stompClient.send(base_path + "notifications", {},
```

Comment:
Following DRY ("dont-repeat-yourself"), we could extract the building of the path into a method.